showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITBidiStreaming.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.showcase.v1beta1.it;
+
+import com.google.api.core.SettableApiFuture;
+import com.google.api.gax.rpc.ClientStream;
+import com.google.api.gax.rpc.ResponseObserver;
+import com.google.api.gax.rpc.StreamController;
+import com.google.common.truth.Truth;
+import com.google.showcase.v1beta1.EchoClient;
+import com.google.showcase.v1beta1.EchoRequest;
+import com.google.showcase.v1beta1.EchoResponse;
+import com.google.showcase.v1beta1.it.util.TestClientInitializer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ITBidiStreaming {
+
+  private EchoClient grpcClient;
+
+  @Before
+  public void setUp() throws Exception {
+    // Create gRPC Echo Client
+    grpcClient = TestClientInitializer.createGrpcEchoClient();
+  }
+
+  // The current implementation of BIDI streaming on Echo showcase server is that it would echo the
+  // request content back on every request, so this test verifies that the response content is
+  // exactly the same as request content.
+  // Ideally we should make the BIDI streaming server more generic, e.g. only respond when there are
+  // three requests, respond twice for every request etc. If that happens, the response content may
+  // not be exactly the same as request content.
+  @Test
+  public void testGrpc_splitCall_shouldListensToResponse() throws Exception {
+    // given
```

Comment:
nit: The spacing you've added between the `arrange`, `act` and `assert` blocks makes the separation pretty clear so we may not need this comment. Same below.

Reply:
Thanks for the suggestion! I think this would still help other developers to read the tests so I'll keep it.